lib/glific/flows/action.ex

```diff
@@ -476,6 +484,50 @@ defmodule Glific.Flows.Action do
     end
   end
 
+  def execute(
+        %{type: "set_contact_profile", profile_type: "Create Profile"} = action,
+        context,
+        _messages
+      ) do
+    attrs = %{
+      name: ContactField.parse_contact_field_value(context, action.value["name"]),
+      type: ContactField.parse_contact_field_value(context, action.value["type"]),
+      contact_id: context.contact.id,
+      language_id: context.contact.language_id,
```

Comment:
Can we call a function in Profile with name handle_flow_action(context, action, type)

It will help to consolidate everything on the profile file.